app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Constraints.scala

```diff
@@ -116,4 +116,10 @@ trait Constraints {
       case invalid => invalid
     }
   }
+
+  protected def checkCorrectLength(length: Int, invalidKey: String): Constraint[String] = Constraint {
```

Comment:
```suggestion
  protected def exactLength(length: Int, invalidKey: String): Constraint[String] = Constraint {
```

please can you also add unit tests in `ConstraintsSpec`

Reply:
Added.